components/script/dom/node.rs

```diff
@@ -299,6 +299,36 @@ impl Node {
         }
     }
 
+    /// Clean up flags and unbind from tree.
+    pub fn complete_remove_child(&self, child: &Node, context: UnbindContext) {
```

Comment:
This doesn't use `self` at all right? Maybe `fn complete_remove_subtree(root: &Node, context: &UnbindContext)`?